### PR TITLE
Fixed nav bar and logo and updated the website responsiveness

### LINKED ISSUE #33: Website Responsiveness  and Design
## What UI bug/error you faced? Could you describe?
The Website is not properly responsive
 I want to work to work on this issue 
please assign me for this issue

### PR DESCRIPTION
**Pull request by Avishake007**
### Pull Request

This PR fixes  #33 

1. What type of change does your PR bring? 
    - [x] Bug fix
    - [x]  Issue fix

### CODE REVIEW

**Comment by Avishake007** [comment]
@alphaX86 @Srimathij  could you please tell me why its showing all checks failed and where I am doing wrong.

**Comment by alphaX86** [comment]
Don't bother the checks @Avishake007 I'm fixing that....

File: .github/workflows/label.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ name: Labeler
 on: [pull_request]
 
 jobs:
-  label:
+  label: label2
```
Comment (alphaX86) [inline .github/workflows/label.yml]: Neglect those changes. Please revert them

**Review by alphaX86 — CHANGES_REQUESTED** [review]

File: .github/workflows/label.yml (hunk 1 above)
Comment (alphaX86) [reply .github/workflows/label.yml]: Do that change alone, that's enough. I didn't say you to make this change. I'm still trying to fix them

**Comment by alphaX86** [comment]
One sec.. reviewing...

**Comment by alphaX86** [comment]
@Avishake007 the responsiveness is perfect! But there's an issue now, I guess it's due to recent changes

**Comment by alphaX86** [comment]
image

Above the image there are actually links and icons below the profiles, it seems a bit messed up

**Review by alphaX86 — CHANGES_REQUESTED** [review]
Could you make those changes I suggested?

**Comment by alphaX86** [comment]
Also you may need to refine a bit.... as below....
image


I'll send some PRs for ref

**Comment by alphaX86** [comment]
Refer PRs #40 #37 for idea to remake

**Comment by alphaX86** [comment]
The site now works as expected! Great work @Avishake007 👏

**Review by alphaX86 — APPROVED** [review]
LGTM! Good to go

**Pull request merged**